src/styles/_media.scss

```diff
@@ -0,0 +1,5 @@
+@media (min-width: 1024px) {
+  .block {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}

```

Comment:
Don't put media queries to separate file. SASS allows you to write it inside corresponding selector.